### ISSUE TITLE
Protobuff error when using find_label_issues

### ISSUE THREAD

**Issue by gordon-lim** [cleanlab/cleanlab#495]
Code used 
```
labels = np.arange(16)
model = models.vgg16(pretrained=True)
model.classifier[6] = torch.nn.Linear(in_features=4096, out_features=16, bias=True)
model.load_state_dict(torch.load("xxx"))
df = pd.read_json('zzz')
# Option 2 - works with ANY ML model - just input the model's predicted probabilities
ordered_label_issues = find_label_issues(
    labels=labels,
    pred_probs=df['logits'].tolist(),  # out-of-sample predicted probabilities from any model
    return_indices_ranked_by='self_confidence',
)
```

Error:
```
[libprotobuf FATAL google/protobuf/stubs/common.cc:83] This program was compiled against version 3.9.2 of the Protocol Buffer runtime library, which is not compatible with the installed version (3.20.1).  Contact the program author for an update.  If you compiled the program yourself, make sure that your headers are from the same version of Protocol Buffers as your link-time library.  (Version verification failed in "bazel-out/k8-opt/bin/tensorflow/core/framework/tensor_shape.pb.cc".)
terminate called after throwing an instance of 'google::protobuf::FatalException'
  what():  This program was compiled against version 3.9.2 of the Protocol Buffer runtime library, which is not compatible with the installed version (3.20.1).  Contact the program author for an update.  If you compiled the program yourself, make sure that your headers are from the same version of Protocol Buffers as your link-time library.  (Version verification failed in "bazel-out/k8-opt/bin/tensorflow/core/framework/tensor_shape.pb.cc".)
Aborted
```

**Comment by jwmueller** [comment]
How are your `labels` formatted?  Could you show what your df labels column looks like, as well as which version of cleanlab you are running? And could you do the same for `pred_probs`? 

You can also check each of the requirements listed here regarding the format of `labels`:  https://docs.cleanlab.ai/stable/tutorials/faq.html#How-do-I-format-classification-labels-for-cleanlab?

Make sure pred_probs is a 2D numpy array as well.

You could also try setting the `n_jobs` arg = 1 just in case that's causing any problems. However your bug seems strange to me since cleanlab is not using protobuf in any way, so I'd guess the error is due to how you formatted labels or pred_probs. It's also unclear to me how you are using the `model` since it does not seem to be invoked in your example code.

**Comment by elisno** [comment]
Hi @gordon-lim. Thanks for the issue!

To improve the odds of getting help from others on your issue it's important that you create a Minimum Working Example that reproduces the issue. 

- Right now, I can't run your snippet as the relevant imports are missing (cleanlab, numpy, pandas, torchvision?)
- Note that you don't have to provide your `model` to `find_label_issues` in your MWE, so there's no reason to show it in your pseudocode.
- Try to use dummy data instead fetching unavailable/private data in your code with `pd.read_json('zzz')`?

## Some questions based on your code:

1. How many _examples_ are in the dataset that you want to call `find_label_issues` on, i.e. how many rows are in `df`?
2. How many distinct classes are in your dataset? It looks like not mixing those up with the `labels` vector whose length should match the number of rows in `df`, i.e. each example should have a label that can be any one of the classes (for multi-class classification).
3. Your `pred_probs` should be a 2D numpy array, which I doubt you can get with `df['logits'].tolist()`.

You may want to go over the [docs](https://docs.cleanlab.ai/master/cleanlab/filter.html#cleanlab.filter.find_label_issues) for `cleanlab.filter.find_label_issues` to get the arguments properly formatted.

## Some questions based on the error:

These issues with runtime version are hard to debug without better info on your environment.
What hardware, OS, Python versions, cleanlab version, etc. are you running?
